source/css/_common/components/tags/tags.styl

```diff
@@ -1,7 +1,7 @@
 @import "full-image";
 @import "blockquote-center";
 @import "group-pictures";
+@import "label";
 @import "note" if hexo-config('note.style') == 'simple' || hexo-config('note.style') == 'flat';
 @import "note-modern" if hexo-config('note.style') == 'modern';
-@import "label" if hexo-config('label');
```

Comment:
It seems need to remove `label` option from config.

Reply:
Yes, it's related to https://github.com/theme-next/theme-next.org/pull/18, and won't cause any side effect. The `label` option only appears once. Considering that we didn't add options for most tags (e.g. button, center-quote, full-image etc.), it would be better to remove it.

Reply:
So?

Reply:
Updated, looks fine now

Reply:
@ivan-nginx Able to merge now?